src/settings/appsettings.cpp

```diff
@@ -36,6 +36,12 @@
 #include <QTextStream>
 #include <QTranslator>
 
+#ifdef Q_OS_DARWIN
+#define I18N_DIR "."
+#else
+#define I18N_DIR "translations"
+#endif
```

Comment:
Could you put it inside `AppSettings::applyLocale` as a regular `QStringLiteral` variable?